code/modules/crafting/guncrafting.dm

```diff
@@ -104,6 +104,10 @@
 	desc = "A suitcase containing the necessary gun parts to build a full gun, when combined with a gun kit. Use it directly on a gunkit to rapidly assemble it."
 	icon_state = "syndicase"
 
+/obj/item/weaponcrafting/gunkit/universal_gun_kit/sol_gov
+	name = "\improper sol gov universal self assembling gun parts kit"
+	icon_state = "totallynotikea"
```

Comment:
totally not ikea?

Reply:
colouring of the self assembling kit is the colour of the ikea colour pallet. Can be changed

Reply:
Discovering new colors every day 0-0
I would change it to "solgovcase" and turn this joke into a comment.

Reply:
renamed to solcase